webclient/src/variables.scss

```
@@ -9,26 +11,27 @@ $theme-accent: #0065bd;  // TUM blue
 // to be declared outside an @if scope.
 // -- Custom vars
 $header-color: #fff;
-$header-shadow-color: rgba(0,0,0,.1);
-$autocomplete-box-shadow: 0 .05rem .2rem rgba(48,55,66,.3);
+$header-shadow-color: rgba(0, 0, 0, 10%);
+$autocomplete-box-shadow: 0 .05rem .2rem rgba(48, 55, 66, 30%);
 $autocomplete-comment-color: #777;
 $autocomplete-filter-bg: #f8f8f8;
 $autocomplete-filter-text: $autocomplete-comment-color;
 $footer-color: #fafafa;
 $footer-setting-bg-disabled: #999;
-$feedback-overlay-bg: rgba(247,248,249,.75);
-$feedback-box-shadow: 0 .2rem .5rem rgba(48,55,66,.3);
-$toast-btn-bg: #0002;
-$toast-btn-bg-hover: #0003;
-$toast-btn-bg-active: #0004;
+$feedback-overlay-bg: rgba(247, 248, 249, 75%);
+$feedback-box-shadow: 0 .2rem .5rem rgba(48, 55, 66, 30%);
+$toast-btn-bg: rgba(0, 0, 0, 13%);
+$toast-btn-bg-hover: rgba(0, 0, 0, 2%);
+$toast-btn-bg-active: rgba(0, 0, 0, 27%);
 $card-highlighted-bg: #f7f7f7;
 $text-gray: #aaa;
-$border-default: #e1e1e1;
+$border-color: #e1e1e1;
```

`border-color` is a spectre.css variable, we shouldn't shadow it here.
```suggestion
$border-color-default: #e1e1e1;
```